web/src/lib/components/assets/thumbnail/thumbnail.svelte

```diff
@@ -114,6 +116,11 @@
 					</div>
 				{/if}
 
+				{#if showArchiveIcon && asset.isArchived}
+					<div class="absolute {asset.isFavorite ? 'bottom-10' : 'bottom-2'} left-2 z-10">
+						<ArchiveArrowDownOutline size="24" class="text-white" />
+					</div>
+				{/if}
```

Comment:
Haven't looked into what's causing this yet, but the icons can sometimes show incorrectly:
1. Favorite and archive an asset and the icons show on top of each other
2. Have an asset that is favorited + archived (reload page), then remove from favorites and the archived icon doesn't properly align

I think the icons look better stacked horizontally, but that's a preference thing

Reply:
Yeah, I can reproduce, look like the class isn't updated correctly

Reply:
resolved by updating the state of the assets that get passed in GalleryViewer